tests/integration/security/rbac/rbac_v2_test.go

```diff
@@ -38,23 +40,24 @@ const (
 )
 
 var (
-	inst                 istio.Instance
-	successIfAuthEnabled bool
+	inst          istio.Instance
+	isMtlsEnabled bool
 )
 
 func TestMain(m *testing.M) {
-	framework.Main("rbac_v2_test", m, istio.SetupOnKube(&inst, setupConfig))
+	framework.
+		NewSuite("rbac_v2", m).
+		// TODO(pitlv2109: Turn on the presubmit label once the test is stable.
```

Comment:
After this PR is merged, where will it run?

Reply:
Part of this https://testgrid.k8s.io/istio-presubmits#integration-k8s-tests, but not `circleci-integration-kubernetes`
Also, the test framework will panic if we turn on presubmit label and use testing apps, e.g. a, b, etc. at the moment.